### DISCUSSION TITLE
Image reading issue

### DISCUSSION THREAD

**Discussion by shenao1995** [Project-MONAI/tutorials#1559] [Q&A]
Why does Monai always display a mirrored and rotated image after reading data. And it's different from the original image, which makes me very confused.
**Origin**
![image](https://github.com/Project-MONAI/tutorials/assets/28835780/caee1f11-711b-4c41-a299-87a2aaf76549)
**After Loading**
![image](https://github.com/Project-MONAI/tutorials/assets/28835780/fde4bc03-d175-4d87-a6d0-d339c3c4386d)
Perhaps it's a problem with the coordinate system? Could it be caused by the nibabel library?

**Comment by wyli** [comment]
the image is transposed by default https://github.com/Project-MONAI/MONAI/issues/4862, you can set the reader option to get the other version: `img = LoadImage(image_only=True, ensure_channel_first=True, reader="PILReader", reverse_indexing=True)(filename)` if the PILReader is used.

**Reply by shenao1995** [reply]
Thank you!
